client/src/lib/fetch.js

```diff
@@ -9,6 +9,55 @@ export async function getUserData(id) {
   return { res, resJSON: await res.json() };
 }
 
-export const doOtherThing = () => {
+export function blah() {
   return null;
-};
+}
+
+export async function createMeetup({
+  title,
+  time,
+  meetupType,
+  meetupURL,
+  meetupLocation,
+  groupCount,
+  description,
+  tags,
+}) {
+  const createMeetupEndpoint = `${SERVER_URL}/meetup`;
+
+  let location;
+  if (meetupType === IN_PERSON) {
+    location = {
```

Comment:
Can you also include the name of the location? It should be available as a part of meetupLocation.